### ISSUE TITLE
conda installation fails

### ISSUE THREAD

**Issue by ysBach** [astropy/photutils#1123]
For some reason, I had to re-install Anaconda.
Then I was installing photutils and got the following errors:

```
$ conda install photutils -c astropy

Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: /
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions
```
(macOS 10.14.6, MBP 2018 15", Most recent Anaconda3 with Python 3.8)

I am not sure if it is because there're something remaining on my laptop or photutils problem.
FYI, pip install photutils successfully installed photutils.

**Comment by larrybradley** [comment]
@ysBach Thanks for reporting this.  I get the same error using Python 3.8 (but not 3.7).  I found that the installation works fine using the `conda-forge` channel:   `conda install photutils -c conda-forge`.  That's a bit puzzling because the astropy channel simply mirrors the conda-forge channel.  Any ideas @mwcraig, @bsipocz?

**Comment by mwcraig** [comment]
@larrybradley -- can you please share the steps it to took to reproduce the error?

**Comment by larrybradley** [comment]
```
conda create -n test_photutils python=3.8
conda activate test_photutils
conda install photutils -c astropy
```

**Comment by larrybradley** [comment]
on OSX....

**Comment by mwcraig** [comment]
It seems to be working for me now -- the astropy channel stopped copying form conda-forge for a few days when travis stopped, maybe that was the issue?

**Comment by larrybradley** [comment]
It's still broken for me.  I tried with a fresh conda install too.

I did some investigating, and it appears there are a few issues going on.  First, `photutils` requires `gwcs`, which requires `astropy >=4.1`.  However, `astropy 4.0.2` is the latest available version in the main conda channel (https://anaconda.org/anaconda/astropy).

Also `gwcs` requires `asdf`, but `conda install asdf -c astropy` also fails with the same error above. (but it installs fine with `-c conda-forge`.

The bottom line is there appears to be older/incompatible dependency versions in the main anaconda channel, but not with `conda-forge`.   I recommend using `conda install photutils -c conda-forge` to get the latest dependencies.

**Comment by larrybradley** [comment]
@mwcraig I'm wondering why it worked for you -- do you have channels set in your `~/.condarc`?  I'm using an empty `.condarc`.

**Comment by mwcraig** [comment]
Yeah, it is definitely because of my `.condarc`, which contains conda-forge. I'll try with an empty one and see what happens. I would have thought that the older versions of `astropy` from defaults wouldn't be an issue since 4.1 is available in the `astropy` channel but apparently it is. I'll report back within a couple hours.

**Comment by larrybradley** [comment]
The good news is that the `photutils` conda package (in both the `conda-forge` and `astropy` channels) appears fine because `conda install photutils -c astropy` works after `conda install photutils -c conda-forge`.

So, it's just a package dependency issue in conda, which I've given up trying to understand.  :smile:

**Comment by larrybradley** [comment]
@mwcraig The `astropy 4.1` issue may be a red herring, but that's one thing I noticed.  If it helps, the other dependency issues seemed to revolved around `gwcs` and `asdf` using the `astropy` channel.

**Comment by mwcraig** [comment]
This is now fixed. The problem was that `semantic_version` is a dependency of `asdf` that had never been copied over to the astropy channel. For a while that didn't matter because it was on `defaults` with version `2.6`. The most recent `asdf` bumped that the `2.8` which was on `conda-forge` but not defaults.

I've added `semantic_version` to the astropy channel and installing `photutils` from `astropy` now works. [Once I removed my `.condarc` I was able to reproduce].

**Comment by larrybradley** [comment]
Great work, @mwcraig!  🎉 🚀   Many thanks!  I've confirmed that `conda install photutils -c astropy` works again, so closing this issue.
